fsm:
  init_takeoff: 1.5
  lidar_avoidance_distance: 1.0
  global_pose_x_min: -100.0
  global_pose_y_min: -100.0
  global_pose_z_min: 0.0
  global_pose_x_max: 100.0
  global_pose_y_max: 100.0
  global_pose_z_max: 3.0

d0: 2.0
k0: 0.1
thrust_scale: 0.035
manual_trj_switch: false
target_x: 0.0
target_y: 0.0
target_z: 0.0
target_yaw: 0.0
verbos: false